nevergrad/examples/powersystem.py

```diff
@@ -23,12 +28,11 @@
 #     num_years: int = 1,  # Number of years.
 #     failure_cost: float = 500.,  # Cost of not satisfying the demand. Equivalent to an expensive infinite capacity thermal plant.
 
-                    power_system_loss = PowerSystem(num_dams=num_dams, depth=depth, width=width, year_to_day_ratio=year_to_day_ratio, back_to_normal=back_to_normal, num_thermal_plants=num_thermal_plants, constant_to_year_ratio=constant_to_year_ratio) 
-                    optimizer = optimizerlib.SplitOptimizer9(instrumentation=power_system_loss.dimension, budget=budget, num_workers=10) 
```

Comment:
@teytaud Did you succeed in making this work??? `SplitOptimizer9` does not exist if I'm correct, it's supposed to be `SplitOptimizer`